Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+precommit: FORCE tidy lint csv
+
+csv: FORCE
+	go run . csv github.com/google/go-licenses
+
+lint: FORCE
+# There's no need to lint third_party packages, so we explicitly list other packages here.
+	golint --set_exit_status . licenses/...
+
+tidy: FORCE
+	go mod tidy
```

Comment:
If this isn't something that's handled by golangci-lint already, this would be great to add to our actions workflow!

Reply:
Sure, I tried to add it to github actions.

Reply:
I copied the implementation from https://github.com/kubeflow/pipelines/blob/32004ab214478451c875ba3dd520ea730503f3bf/test/presubmit-backend-test.sh#L33-L34, but I cannot trigger github actions before your review to verify this works. Can you give me some sort of permission to let me run github actions automatically?